AI-Upscaling-With-NPU/tools/install_pixi.sh

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env bash
+set -euo pipefail
+# Version: v0.34.0
+
+__wrap__() {
+
+VERSION="${PIXI_VERSION:-latest}"
```

Comment:
Same issue as in install_pixi.ps1:
If we want to use just the particular verified version of PIXI I wonder if leaving the default 'latest' option is safe. I mean if anyone will make changes to the global variable that specifies the PIXI_VERSION  or delete it (for some reasons) we will be installing the latest version of PIXI which was not verified and checked. I wonder if it wouldn't be better to throw an error when version isn't specified instead of using the default 'latest' version.

Reply:
This has been changed to throw an error.